pkg/utils/volumeUtils.go

```diff
@@ -6,8 +6,14 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
+var (
+	// hostpathTypeFile represents the hostpath type
+    hostpathTypeFile = corev1.HostPathFile
+)
+
+
 // CreateVolumeBuilder build Volume needed in execution of experiments
-func CreateVolumeBuilder(configMaps []v1alpha1.ConfigMap, secrets []v1alpha1.Secret) []*volume.Builder {
+func CreateVolumeBuilders(configMaps []v1alpha1.ConfigMap, secrets []v1alpha1.Secret, hostFileVolumes []v1alpha1.HostFile) []*volume.Builder {
```

Comment:
The function name should be `CreateVolumesBuilder`

Reply:
Or this is also fine, Only need to change the comment also

Reply:
This name changed at all occurrences so won't get any error, If you want to sustain this new name. we can update the name inside comment as well otherwise, we will get lint warning.

Reply:
done!